news/2025/2025-04-24-osutaiko-world-cup-finals-recap.md

```diff
@@ -0,0 +1,91 @@
+
+Taiwan's ban was interestingly [BilliumMoto - Sentinel](https://osu.ppy.sh/beatmapsets/2354654#taiko/5069626) (HR2) despite their strong history on the fast HR slot, likely a result of Chile's intimidating 3.06 million team score in their previous match; Chile responded by banning [Ludicin - Signal](https://osu.ppy.sh/beatmapsets/2354876#taiko/5070177) (NM6) into the very tech-oriented Taiwan squad, followed by a pick of [cosMo@Bousou-P feat. HARU - LowSpeed Fake Lyric](https://osu.ppy.sh/beatmapsets/2354609#taiko/5069510) (FM2)… that led to a quick break point from Taiwan! ::{ flag=CL }:: [ulko](https://osu.ppy.sh/users/1263669)'s 99.6% HD full combo on the brutal antimod wasn’t enough to hold the pick, but ::{ flag=CL }:: [chibamasu](https://osu.ppy.sh/users/16067522)'s 99.48% run on [TOMOYU & Supire feat. RIHINA - greenREgeneration](https://osu.ppy.sh/beatmapsets/2354626#taiko/5069549) (HD1) stood tall above the rest to put Chile on the board with a counter break on Taiwan's first pick.
+
+After staving off a huge second half comeback to take Chile's second pick of [cosMo@BousouP feat. Hatsune Miku - DeroDeroDeroDe](https://osu.ppy.sh/beatmapsets/2317287#taiko/4962143) (HD2), Taiwan finally broke the break streak with a dominant win on [hugepulse - Kaitei Gensou Toshi](https://osu.ppy.sh/beatmapsets/1955875#taiko/5069532) (NM7), with ::{ flag=TW }:: [HarryAda](https://osu.ppy.sh/users/12892087) narrowly missing out on a 99.5% full combo that would give the winner’s bracket a run for their money! Chile held on with three amazing HDHR scores on [Umeboshi Chazuke - Galaxy in Toybox](https://osu.ppy.sh/beatmapsets/559156#taiko/1182564) (FM1) to make it 2-3, Taiwan printed a point from the polyrhythmic [KERO - Color Printer](https://osu.ppy.sh/beatmapsets/2167213#taiko/4572334) (HR1), and Chile kept up on [KNOWER - Time Traveler](https://osu.ppy.sh/beatmapsets/2354840#taiko/5070071) (DT1) by way of not one but three full-combo near misses on one of the week's toughest maps.
+
+Now up by a score of 3-4, Taiwan seemingly found an extra gear that Chile couldn't quite seem to match. Two full combos on [Spire of Lazarus - Big Hat Logan](https://osu.ppy.sh/beatmapsets/2354620#taiko/5069531) (NM5) with an unreal 3x100 from ::{ flag=TW }:: [HarryAda](https://osu.ppy.sh/users/12892087), three scores with one combined miss on the highly stressful [Amanogawa Converter - Never play me](https://osu.ppy.sh/beatmapsets/2354618#taiko/5069528) (NM4) for one more break point, and one tightly contested, high-accuracy battle on [Rahatt - Gunoi Chop](https://osu.ppy.sh/beatmapsets/2354610#taiko/5069511) (NM1) later, team Taiwan had done it.
+
+With a huge 7-3 win over a terrifying opponent in Chile, who still deserve huge congratulations for their best finish ever at 4th place, Taiwan came out on top of the brutal Finals lower bracket, advancing to the Grand Finals.
```

Comment:
"who" is generally for people
"Chile deserve" is also weird because the country is grammatically not plural

```suggestion
With a huge 7–3 win over a terrifying opponent in Chile, a team that still deserves huge congratulations for their best finish ever at 4th place, Taiwan came out on top of the brutal Finals lower bracket, advancing to the Grand Finals.
```